.github/labeler.yml

```
@@ -48,6 +48,11 @@
 "Bionics":
   - "data/json/bionics.json"
 
+"Map / Mapgen":
+  - mapgen/**/*
+  - mapgen_palettes/**/*
+  - overmap/**/*
```

```suggestion
  - "**/mapgen/**/*"
  - "**/mapgen_palettes/**/*"
  - "**/overmap/**/*"
```
I recall thinking it would be good for this to just detect changes in specifically named folders, like here, to detect _any_ nested mapgen folder
I wanted to test it back when you added npcs and monsters, but forgot about, I got it working here yesterday <https://github.com/casswedson/auto-label-issue-pr/pull/10>
Can you change the monstergroups, itemgroups, monsters, npcs, and recipes use this format

---

Sure thing!